src/services/cron.js

```diff
@@ -29,7 +29,7 @@ const checkServerStatusJob = () =>
     process.env.SERVER_STATUS_CRON_JOB_SCHEDULE,
     async () => {
       try {
-        await axios.get('http://localhost:3000/status');
+        await axios.get(`${process.env.SERVER_BASE_URL}status`);
```

Comment:
Don't forget the `/` between based URL and `status`.

Reply:
sorry :( , auto-correction in VS Code did that  and didn't notice that